clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -158,46 +184,52 @@ func (c *clusterClient) GetMachineObjects() ([]*clusterv1.Machine, error) {
 	return machines, nil
 }
 
+func (c *clusterClient) GetMachineObjects() ([]*clusterv1.Machine, error) {
+	// TODO: Iterate over all namespaces where we could have Cluster API Objects https://github.com/kubernetes-sigs/cluster-api/issues/252
+	return c.GetMachineObjectsInNamespace(apiv1.NamespaceDefault)
+}
+
 func (c *clusterClient) CreateClusterObject(cluster *clusterv1.Cluster) error {
-	// TODO: Support specific namespaces https://github.com/kubernetes-sigs/cluster-api/issues/252
-	_, err := c.clientSet.ClusterV1alpha1().Clusters(apiv1.NamespaceDefault).Create(cluster)
+	namespace := apiv1.NamespaceDefault
```

Comment:
Instead of using apiv1.NamespaceDefault, should this rather use the namespace associated with the current kubeconfig context instead? That is the behavior I would expect, especially if passing in an existing kubeconfig for an external cluster.

Reply:
Good suggestion.